### DISCUSSION TITLE
X-ray glasses

### DISCUSSION THREAD

**Discussion by anchillabc** [littlesera/LSCG#468] [Ideas]
Sure, we have the spell effect and the lead lined clothes, but what about x-ray glasses?

Basically, a keyword (x-ray, xray, x ray) for items like the interactive visor, the groucho marx glasses, smooth latex mask or any kind of eye restraint one can see through which, when the crafted item is equipped, gives the wearer x-ray vision until removed.

**Comment by littlesera** [comment]
Unable to do groucho marx or other 'wardrobe' glasses as they can't be crafted. But implemented for:
"InteractiveVisor",
"InteractiveVRHeadset",
"FuturisticMask",
"BlackoutLenses",
"DroneMask",
"AnimeLenses"

**Reply by littlesera** [reply]
Implemented in v0.3.45
